alexandria/settings/alexandria.py

```diff
@@ -169,8 +169,6 @@ def default(default_dev=env.NOTSET, default_prod=env.NOTSET):
 ALEXANDRIA_ENABLE_CHECKSUM = env.bool("ALEXANDRIA_ENABLE_CHECKSUM", default=True)
 
 # Clamav service
-CLAMD_SOCKET = env.str("ALEXANDRIA_CLAMD_SOCKET", default="/var/run/clamav/clamd.ctl")
-CLAMD_USE_TCP = env.bool("ALEXANDRIA_CLAMD_USE_TCP", default=True)
 CLAMD_TCP_SOCKET = env.str("ALEXANDRIA_CLAMD_TCP_SOCKET", default=3310)
 CLAMD_TCP_ADDR = env.str("ALEXANDRIA_CLAMD_TCP_ADDR", default="localhost")
 CLAMD_ENABLED = env.bool("ALEXANDRIA_CLAMD_ENABLED", default=False)
```

Comment:
I think since the code is inlined now, we could avoid this unexpected name mismatch between env and settings..

```suggestion
ALEXANDRIA_CLAMD_TCP_SOCKET = env.str("ALEXANDRIA_CLAMD_TCP_SOCKET", default=3310)
ALEXANDRIA_CLAMD_TCP_ADDR = env.str("ALEXANDRIA_CLAMD_TCP_ADDR", default="localhost")
ALEXANDRIA_CLAMD_ENABLED = env.bool("ALEXANDRIA_CLAMD_ENABLED", default=False)
```

Reply:
Done